### DISCUSSION TITLE
Can guests change their order at any time?

### DISCUSSION THREAD

**Discussion by nilskamm** [pretix/pretix#2518] [Q&A] [answered]
Hey! 
I tested some weeks ago if everyone can change their order details like the name on the ticket or their address.
When i tested it with an guest order the "Change details" link did not appear when i opened the order details with the link from the e-mail.

Today i could, change the details before and after payment from a guest order.

How should it normaly be? 
Is there a state or age of the order where a guest cannot change this anymore?

Thanks a lot!

**Comment by raphaelm** [comment, marked as answer]
Yes, there are multiple factors here, for example guests cannot change canceled orders. Also, there's a "last date of modifications" in the settings.

**Comment by nilskamm** [comment]
Thank you very much for your fast response!
I can not check again, why i couldn´t change the details, when i checked it the last time. 
Maybe one of your examples was the reason.
You said that these are examples. Are there more reason why a guest cannot change their order when they have the correct link for their order?
